ansible/host_vars/localhost

```diff
@@ -91,10 +91,10 @@ specs: []
 #     custom_specs: {}
 nova_flavors: []
 
-# The Glance UUID of the image to use for the deployment kernel.
-deploy_kernel_uuid:
-# The Glance UUID of the image to use for the deployment ramdisk.
-deploy_ramdisk_uuid:
+# The Glance name or UUID of the image to use for the deployment kernel.
+deploy_kernel:
+# The Glance name or UUID of the image to use for the deployment ramdisk.
+deploy_ramdisk:
```

Comment:
Yes, that's a good plan. Best to leave their defaults as `none` in host_vars/localhost but it would be nice to put the variables in override.yml.example.